src/translations/sv.json

```diff
@@ -325,6 +322,24 @@
   "page.winter_storage.notice.paragraph1": "Ansökan om vinteruppläggningsplatser för säsongen 2020–2021 är nu aktuell. Vi skickar en faktura till de kunder som har haft rutplats förra säsongen. Om du är ny sökande, ansök om rutplats genom att göra en elektronisk ansökan på webben på <a href='https://venepaikat.hel.fi/sv'>venepaikat.hel.fi/sv</a>. Lediga rutplatser delas ut på hösten i ansökningsordning. Vi skickar ut erbjudande om plats via e-post.",
   "page.winter_storage.notice.paragraph2": "Vi skickar inte någon faktura till de kunder som har haft vinteruppläggningsplats i upptagningsordningsområdet, utan betalning sker till båtplatsreserveringens konto efter att båten har tagits till upptagningsordningsområdet. Vi informerar senare på webben om betalning för plats i upptagningsordningsområdet på adressen <a href='https://hel.fi/båtliv'>hel.fi/båtliv</a>, samt på upptagningsordningsområdets infotavlor. Till de kunder som har betalat för en vinteruppläggningsplats skickas en vinteruppläggningsdekal till hemadressen, Vi ber er klistra dekalen på en synlig plats på båten.",
   "page.winter_storage.notice.paragraph3": "Båtägarna kan ta sin båt till önskat upptagningsordningsområde från och med den 15 september 2020. Observera att man inte får ta båtar till upptagningsordningsområdena innan uppläggningstiden börjar den 15 september 2020.",
-  "page.not_found.info_text": "Sidan du sökte kunde tyvärr inte hittas.",
-  "page.not_found.to_front_page": "Till förstasidan"
+
+  "page.application_sent.message.paragraph1": "Du får snart ett bekräftelsemeddelande till den e-postadress du uppgett. Om du vill ändra eller annullera din ansökan eller undrar om något som gäller ansökan, skicka e-post till venepaikkavaraukset@hel.fi.",
+  "page.application_sent.message.paragraph2": "Helsingfors stads båtplatsreservering<br/>venepaikkavaraukset.hel.fi<br/>Telefonrådgivning må–fr kl. 9–14 på 09 310 87900<br/>Betjäningstid vardagar kl. 9–16 i Tölö Sporthall på Paavo Nurmis gränd 1 E.",
+  "page.application_sent.title": "Tack, din ansökan har skickats!",
+  "page.not_found.message": "Sidan du sökte kunde tyvärr inte hittas.",
+  "page.not_found.title": "404",
+  "page.notification_sent.message.paragraph1": "",
```

Comment:
Should all the missing translations be left out of the files so they fallback to finnish instead? An issue should also be created in jira for providing these missing translations.

Reply:
Created the issue.

It appears that `react-intl` is not capable of falling back to default locale automatically. I could make it so that `sv` and `en` include `fi` for missing values, but let's keep this like it is for now, as I am already in the progress of replacing the translation library in another branch

Reply:
Fair enough 👍